src/azul/service/manifest_service.py

```diff
@@ -1320,30 +1320,26 @@ def write_page_to(self,
                     short_names = project['project_short_name']
                     project_short_names.update(short_names)
                 file_url = self._azul_file_url(file_)
-                # FIXME: The slice is a hotfix. Reconsider.
```

Comment:
If this PR really resolves the issue referenced in the FIXME, the issue would need to be referenced in the commit title. I would also like to see the reasoning behind a claim of resolution.

Reply:
It does not resolve the issue, it only removes one occurrence of it in the code. There is an extant FIXME earlier in the file: https://github.com/DataBiosphere/azul/blob/f251a7934e53e0fc231a1ada066ad3ee1e852dd4/src/azul/service/manifest_service.py#L787

Reply:
The second FIXME was removed because "the slice" was in a line that was deleted.